linera-execution/src/lib.rs

```diff
@@ -370,6 +371,25 @@ pub trait BaseRuntime {
     /// Tests whether a key exists in the key-value store (wait)
     fn contains_key_wait(&mut self, promise: &Self::ContainsKey) -> Result<bool, ExecutionError>;
 
+    /// Tests whether multiple keys exist in the key-value store
+    #[cfg(feature = "test")]
+    fn contains_keys(&mut self, keys: Vec<Vec<u8>>) -> Result<Vec<bool>, ExecutionError> {
+        let promise = self.contains_keys_new(keys)?;
+        self.contains_keys_wait(&promise)
+    }
+
+    /// Tests whether multiple keys exist in the key-value store (new)
```

Comment:
This still says "(new)" and "(wait)". It should explain that it creates and resolves a promise.

Reply:
Ok, done.